app/src/org/commcare/android/tasks/FormRecordCleanupTask.java

```diff
@@ -174,14 +173,17 @@ public static FormRecord getUpdatedRecord(Context context, CommCarePlatform plat
         //NOTE: This does _not_ parse and process the case data. It's only for getting meta information
         //about the entry session.
         TransactionParserFactory factory = new TransactionParserFactory() {
+            public TransactionParser getParser(KXmlParser parser) {
+                String name = parser.getName();
+
+                if (name == null) {
+                    return null;
+                }
 
-            public TransactionParser getParser(String name, String namespace, KXmlParser parser) {
-                if(name == null) { return null;}
-                if("case".equals(name)) {
+                if ("case".equals(name)) {

```

Comment:
All the other 'name' checks use equalsIgnoreCase. Should we change the 'equals' to that in this check?